.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
@job13er 

`eslint-config-frost-standard` has

```
    "ocd/sort-import-declarations": [
      2,
      {
        "localPrefixes": [
          "../",
          "./",
          "dummy/"
        ]
      }
    ],
```

and this PR proposes:

```
    "ocd/sort-import-declarations": [
       1,
       {
         "localPrefixes": [
           "../",
           "./",
           "dummy/",
           "ember-frost-test"
         ]
       }
     ],
```

This "alternative" was established by `eslint-config-frost-standard` and our use of it, and we should be getting our configuration from it, and not from in-repo.